Shopify/hydrogen
# Adding new command to skip verification step for h2 deploy

### WHAT is this pull request doing?

Adding flag to `h2 deploy` to skip the verification step after the deployment has been completed.

### HOW to test your changes?

- `git fetch && git checkout h2-deploy-skip-verification`
- `npm run build`
- `h2 deploy --skip-verification --path=path-to-hydrogen-project`

Should only see "Uploading files" & "Verifying deployment has been completed" but not "Verifying deployment is routable".

#### Post-merge steps

Add new flag to documentation.

#### Checklist
